src/composite-disposable.coffee

```diff
@@ -52,8 +52,9 @@ class CompositeDisposable
   #
   # * `disposable` {Disposable} instance or any object with a `.dispose()`
   #   method.
-  add: (disposable) ->
-    @disposables.add(disposable) unless @disposed
+  add: ->
```

Comment:
I wonder if it might be better to use a [splat](http://coffeescript.org/#splats) here instead of `arguments`?